### DISCUSSION TITLE
basic question about experiment editor

### DISCUSSION THREAD

**Discussion by lemorgan** [NMGRL/pychron#2165] [Q&A] [answered]
I have a really basic question. Do I need to enter aliquot numbers in the experiment editor? I have a bunch of standards (SCTF) to run and can't remember if I need to enter aliquot...or how to make it auto-increment them for me. When I've tried the 'auto-increment identifier' it seems to increment to the next sample instead of the next aliquot?
I know I can add them manually, but it seems like there must be a way to batch edit this and I can't seem to figure it out...
![Screen Shot 2022-04-13 at 9 46 39 AM](https://user-images.githubusercontent.com/18449986/163228795-c76bc9ff-f077-44fa-9067-a9bb8e1c6c64.png)

**Comment by jirhiker** [comment, marked as answer]
***Do I need to enter aliquot numbers in the experiment editor?***
The aliquot should be set only for step heats.  Pychron automatically will determine the aliquot for fusions.
When you add a set of analyses via a "Step Heat Template" Pychron automatically determines the aliquot number by examining the database.  When the step runs pychron  will automatically determine the step label (A, B, C etc)

if you only enter an Identifier (aka L#) e.g. 1000, the first analysis will be 1000-01, the second analyses 1000-02. If you enter 1000 and aliquot 03, then pychron will assume this is a step heat and the first step will have RunID 1000-03A and the second step 1000-03B.  


***When I've tried the 'auto-increment identifier' it seems to increment to the next sample instead of the next aliquot?***
Correct the identifier is the same as L# in mass spec. It is the unique identifier for a position in an irradiation. The aliquot is the number of times an identifier as been analyzed. 


***how do i batch edit?***
Select a set of runs in the table, make the modifications in the Experiment Editor pane
